chain-signatures/contract-sol/programs/chain-signatures-project/src/lib.rs

```diff
@@ -1,16 +1,13 @@
+#![allow(unexpected_cfgs)]
 use anchor_lang::prelude::*;
-use anchor_spl::token::{self, Token};
 
 declare_id!("4kZoBXmUBLveRS3sboGF557tYsR7SzLDsWmP4sz7VQEs");
 
 #[program]
 pub mod chain_signatures_project {
     use super::*;
 
-    pub fn initialize(
-        ctx: Context<Initialize>,
-        signature_deposit: u64,
-    ) -> Result<()> {
```

Comment:
@ailisp I've noticed that the contract here and in the https://github.com/sig-net/chain-signatures-solana-stub repo are not the same. Is that because of the conflicts you have mentioned?

Reply:
updated to latest contract